client/my-sites/marketing/connections/services-group.jsx

```diff
@@ -55,8 +55,9 @@ const SharingServicesGroup = ( { isFetching, services, title } ) => {
 			<ul className="sharing-services-group__services">
 				{ services.length
 					? services.map( service => {
-							const Component = Components.hasOwnProperty( service.ID )
-								? Components[ service.ID ]
+							const componentName = service.ID.replace( /-/g, '_' );
+							const Component = Components.hasOwnProperty( componentName )
+								? Components[ componentName ]
```

Comment:
What about simplifying this into this?
`const Component = Components[ componentName ] || Service;`

Reply:
Good point! I think the ternary statement was there before, but let's take the opportunity to simplify it.